in_toto/gpg/util.py

```diff
@@ -87,25 +87,24 @@ def hash_object(headers, algorithm, content):
   return hasher.finalize()
 
 
-def parse_packet_header(data, expected_type):
+def parse_packet_header(data, expected_type=None):
   """
   <Purpose>
-    Parse an RFC4880 packet header and return its payload
+    Parse an RFC4880 packet header and return its payload, length and type.
 
   <Arguments>
```

Comment:
`expected_type` is not listed under `<Arguments>`.

Reply:
Added in 93b78d7